webhook.py

```diff
@@ -48,46 +56,31 @@ def get_evaluation(board, time_limit):
     score = score.score()
 
     if mate is not None:
+        # if the evaluation is a mate, format it as '#<number of moves>'
         return '#{:+d}'.format(mate), next_move
     else:
+        # if the evaluation is not a mate, format it as '+/-<score>'
         return '{:+.2f}'.format(score / 100), next_move
 
+# function to send a Discord webhook with the current board state and evaluation
 def send_webhook(png, last_move, pts, move, board):
     webhook_url = os.environ['WEBHOOK_URL']
     webhook = DiscordWebhook(url=webhook_url, username='README Chess', avatar_url='https://github.com/ChessCom.png')
     webhook.add_file(file=png.read(), filename='image.png')
 
+    # create an embed for the webhook with the current evaluation and recommended next move
     embed = DiscordEmbed(title=last_move, description='New move played', color=(0xefefef if board.turn == chess.WHITE else 0x303030))
     embed.set_footer(text='Generated by marcizhu/readme-chess')
     embed.set_timestamp()
     embed.add_embed_field(name='Evaluation', value=pts)
     embed.add_embed_field(name='Next best move', value=move)
     embed.set_image(url='attachment://image.png')
 
+    # add the embed to the webhook and send it
     webhook.add_embed(embed)
     webhook.execute()
 
+# function to define the light blue board style
 def style_lightblue():
     colors = dict()
-    colors['square light'] = '#D6DCE0'
-    colors['square dark'] = '#7A909D'
-    colors['square light lastmove'] = '#B7D288'
-    colors['square dark lastmove'] = '#82A465'
-    return colors
-
-board = load_board(get_last_modified('games'))
-check_pos = board.king(board.turn) if board.is_check() else None
-lastmove = board.peek() if len(board.move_stack) > 0 else None
-svg = chess.svg.board(board=board, lastmove=lastmove, check=check_pos, size=900, colors=style_lightblue())
-png = generate_png(svg)
-
-pts, move = get_evaluation(board, 10.0)
-
-if len(board.move_stack) == 0:
-    send_webhook(png, 'Start new game', pts, move, board)
-else:
-    last_move = board.peek();
-    board.pop()
-
-    last_move = move_to_san(board, last_move)
-    send_webhook(png, last_move, pts, move, board)
+    colors['square
```

Comment:
I think you accidentally deleted the some code... :P

Reply:
Yes. In a hurry, I deleted some of the code mistakenly

Reply:
I'll solve it no problem.